### ISSUE TITLE
How to build expat as a static library?

### ISSUE THREAD

**Issue by tianchao-haohan** [libexpat/libexpat#252]
I found that a so will be created with cmake/make. However, I need to build the libexpat as static library in linux with gcc. Is there anybody know how to build a static library?

**Comment by hartwork** [comment]
Have you tried `-DBUILD_shared=OFF`?

**Comment by tianchao-haohan** [comment]
@hartwork

**Comment by hartwork** [comment]
Please provide details about your full command line to CMake, the OS distribution, compiler, the command causing that error message (e.g. from `make VERBOSE=1`). Thanks!

**Comment by hartwork** [comment]
Closing due to lack of reply. Happy to re-open later.

**Comment by cyrusbehr** [comment]
Looks like the required argument is `-DEXPAT_SHARED_LIBS=OFF`

**Comment by hartwork** [comment]
Yes, the option was renamed from `BUILD_shared` to `EXPAT_SHARED_LIBS` in commit fce015875e6153d417c60818946073f6c6aae692 which is first included in release 2.2.8.

```console
# git tag --contains fce015875 | sort -V
R_2_2_8
R_2_2_9
R_2_2_10
R_2_3_0
R_2_4_0
R_2_4_1
```
